### ISSUE TITLE
popupHeight and favGridCols reset to original default after exit or restart

### ISSUE THREAD

**Issue by maxsspeaker** [Zren/plasma-applet-tiledmenu#129]
for some unknown reason, the menu does not want to read the popupHeight and favGridCols values ​​from the config, which causes the menu size to reset

to these values:
popupHeight: 0 
favGridCols: 200 

but there were such values:
favGridCols 12
popupHeight 400

before logging out:
![2022-05-10_10-57](https://user-images.githubusercontent.com/56259377/167580841-3b867f9c-a982-4f25-8ac5-26e648a38f39.png)

after logging out: 
![2022-05-10_10-58](https://user-images.githubusercontent.com/56259377/167580943-e61d306e-67c3-4b05-9088-55694ce8026e.png)

I have read other Issues but didn't find anything about it.

**Comment by maxsspeaker** [comment]
> `200 columns wide!? Did you mix up the two?`

yes, I mixed up the numbers when I rewrote them from the config 

I have Kde Neon, and from what I read, this bug will be fixed on May 14th, but is there any way to fix it right away?

tell me please, I'm not very good at configurations and sources...

**Comment by Zren** [comment]
You can use `Meta+RightClick+Drag` to resize a window as a workaround. You'll need to do it every restart.

If you restart multiple times a day... then I guess you could:

Edit: `~/.local/share/plasma/plasmoids/com.github.zren.tiledmenu/contents/ui/Main.qml`

Change:

```qml
	Layout.minimumWidth: config.leftSectionWidth
	Layout.preferredWidth: config.popupWidth
	Layout.preferredHeight: config.popupHeight
```

to:

```qml
	Layout.minimumWidth: config.popupWidth
	Layout.minimumHeight: config.popupHeight
	Layout.preferredWidth: config.popupWidth
	Layout.preferredHeight: config.popupHeight
```
